src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,15 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
+public class UserService {
+    
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String userInfo : records) {
+            if (userInfo.split(":")[0].equals(email)) {
```

Comment:
let's create String[] userRecord = userInfo.split() 
before checkin condition in if, this way you won't split arr twice when if generates true and it will be more readable